.env.example

```diff
@@ -3,6 +3,9 @@
 # Replace the following with the output from "rake secret"
 APP_SECRET_TOKEN=REPLACE_ME_NOW!
 
+# Replace the following with another output from "rake secret"
+APP_ENCRYPTION_PASSPHRASE=REPLACE_ME_NOW!
```

Comment:
I agree with @dsander, and I am not 100% sure this would actually help to set `APP_SECRET_TOKEN` and `APP_ENCRYPTION_PASSPHRASE` to default to the same.